### ISSUE TITLE
mergeLonghand removes things it shouldn't remove

### ISSUE THREAD

**Issue by hudochenkov** [cssnano/cssnano#554]
`mergeLonghand` removes CSS in `cssnano@4.0.2`.

Input:

```css
.b {
    border-top-color: rgba(85, 85, 85, 0.95);
    border-bottom: none;
}
```

Output:

```css
.b {
  border-bottom: none;
}
```

**Comment by vseguin** [comment]
@hudochenkov Do you have any workaround? Did you downgrade meanwhile?

**Comment by hudochenkov** [comment]
@vseguin I disabled this rule:

```js
require('cssnano')({
    preset: [
        'default',
        {
            mergeLonghand: false,
        },
    ],
}),
```

**Comment by vseguin** [comment]
@hudochenkov  worked for me, thank you so much!

**Comment by andyjansson** [comment]
Looking into this.

**Comment by evilebottnawi** [comment]
Done in https://github.com/cssnano/cssnano/pull/555

**Comment by evilebottnawi** [comment]
Fixed in `cssnano@4.0.3`

**Comment by hudochenkov** [comment]
Thank you for a quick fix!
